fluidBuild.config.cjs

```diff
@@ -38,9 +38,11 @@ module.exports = {
 		"build:genver": [],
 		"typetests:gen": ["^tsc", "build:genver"], // we may reexport type from dependent packages, needs to build them first.
 		"tsc": tscDependsOn,
-		"build:esnext": tscDependsOn,
+		"build:esnext": [...tscDependsOn],
```

Comment:
Is this change needed?

Reply:
Not technically but it makes refactoring the tasks easier. That's what led to it - I was changing the tasks around and this stuck around